src/mongocxx/collection.hpp

```diff
@@ -478,9 +479,26 @@ class MONGOCXX_API collection {
     ///
     void rename(stdx::string_view new_name, bool drop_target_before_rename = false);
 
+    ///
+    /// Sets the read_concern for this collection. Changes will not have any affect on existing
+    /// cursors or other read operations which use the previously-set read concern.
+    ///
+    /// @param rc
+    ///   The new @c read_concern
+    ///
+    /// TODO add link to docs once they exist.
+    ///
+    void read_concern(class read_concern rc);
+
+    ///
+    /// Gets the read_concern for the collection.
+    ///
+    /// @return The current read_concern.
+    ///
+    class read_concern read_concern() const;
 
     ///
-    /// Sets the read_preference for this collection. Changes will not have any effect on existing
+    /// Sets the read_preference for this collection. Changes will not have any affect on existing
```

Comment:
find and replace, at it again... :S
I think it would be better as a verb here anyway, but no sense making the change just to rephrase a comment